Lib/test/test_mmap.py

```diff
@@ -264,6 +264,26 @@ def test_bad_file_desc(self):
         # Try opening a bad file descriptor...
         self.assertRaises(OSError, mmap.mmap, -2, 4096)
 
+    def test_if_crash(self): # test for issue gh-103987
```

Comment:
```suggestion
    def test_if_crash(self):
        # See gh-103987
```

Reply:
Also, I'd like to see more meaningful name for this test-case, but I have no ideas :(
Maybe, `test_unexpected_mmap_close`?

Reply:
sure